### DISCUSSION TITLE
returning-xml-from-minimal-apis-in-dotnet-6/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#131] [Announcements]
# returning-xml-from-minimal-apis-in-dotnet-6/

In this post I show how to return an XML response from minimal APIs, how to create a helper extension method, and discuss performance considerations

https://andrewlock.net/returning-xml-from-minimal-apis-in-dotnet-6/

**Comment by MMarins** [comment]
I just really want to say thank you for the great content. It helped me a lot. Unfortunately, the conversion from Sterling to the Brazilian Real is a bit cruel. 😟

[a comment by an automated account is not shown]

**Reply by andrewlock** [reply]
<em>[Sock](https://disqus.com/by/andrewlockdotnet/) (@andrewlock) commented [on Disqus](https://disqus.com/home/forum/andrewlock/) at July 11 2022, 11:02, in reply to [Oisín G.](https://disqus.com/by/oising/)'s [comment](https://github.com/andrewlock/blog-comments/discussions/131#discussioncomment-6063150)</em>

---

<p>Good call, thanks, didn't think that through 🙂 I updated the code!</p>

[a comment by an automated account is not shown]
